src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -1012,17 +1013,25 @@ fd_exec_vm_syscall_test_run( fd_exec_instr_test_runner_t *          runner,
   effects->stack->size = (uint)FD_VM_STACK_MAX;
   fd_memcpy( effects->stack->bytes, vm->stack, FD_VM_STACK_MAX );
 
-  effects->inputdata = FD_SCRATCH_ALLOC_APPEND(
-    l, alignof(uchar), PB_BYTES_ARRAY_T_ALLOCSIZE( input_data_sz ) );
-  effects->inputdata->size = (uint)input_data_sz;
-  fd_memcpy( effects->inputdata->bytes, vm->input, input_data_sz );
+  if( input_data_sz ) {
```

Comment:
There's a difference between null vs 0-length array.
By making 0-length array null, the encoded pb is the same, so we can do binary comparison (if we want).